pkg/deviceshifu/deviceshifubase/deviceshifubase_config.go

```diff
@@ -119,15 +119,15 @@ func NewDeviceShifuConfig(path string) (*DeviceShifuConfig, error) {
 	if instructions, ok := cfg[ConfigmapInstructionsStr]; ok {
 		err := yaml.Unmarshal([]byte(instructions), &dsc.Instructions)
 		if err != nil {
-			log.Fatalf("Error parsing %v from ConfigMap, error: %v", ConfigmapInstructionsStr, err)
+			klog.Fatalf("Error parsing %v from ConfigMap, error: %v", ConfigmapInstructionsStr, err)
```

Comment:
Do you still need to return err if you fatal here? Or maybe we should avoid using fatal and return error to the callee at every call.

Reply:
bad configmap should be a fatal